dev/com.ibm.ws.ejbcontainer/src/com/ibm/ws/ejbcontainer/osgi/internal/NameSpaceBinderImpl.java

```diff
@@ -151,6 +155,96 @@ public void bindDefaultEJBLocal(EJBBinding bindingObject, HomeRecord hr) {
 
     }
 
+    /**
+     * Binds the simpleBindingName custom binding
+     *
+     * Caller should ensure the simpleBindingName exists
+     *
+     * @param bindingObject - the EJBBinding
+     * @param hr - the bean home record
+     * @param local - is local bean
+     */
+    @Override
+    public void bindSimpleBindingName(EJBBinding bindingObject, HomeRecord hr, boolean local) {
+        final boolean isTraceOn = TraceComponent.isAnyTracingEnabled();
+
+        BeanMetaData bmd = hr.getBeanMetaData();
+        String bindingName = bmd.simpleJndiBindingName;
+
+        // Clean up names with multiple namespaces in front.
+        boolean endsInNamespace = bindingName.endsWith(":");
```

Comment:
The check for ending with ":" seems either unnecessary, as you cannot bind an EJB into a context, or that it should result in an error.

Reply:
Opened #9600 to address code change and updated #9557 (additional err tests) to add more automated tests.